pkg/util/errors/wrapper.go

```diff
@@ -0,0 +1,22 @@
+package errors
+
+import (
+	"fmt"
+	"runtime"
+)
+
+func getCallerInfo() (string, int, string) {
+	pc, file, line, ok := runtime.Caller(2)
+	if !ok {
+		return "?", 0, "?"
+	}
+
+	fn := runtime.FuncForPC(pc)
+	return file, line, fn.Name()
+}
+
+// Wrap allows you to wrap the error message with file,line,caller informations which can be useful for reporting
```

Comment:
seems the comment stopped midway, is that the case?

Reply:
comments have been fixed